scripts/mapd-deps-conda-dev-env.yml

```diff
@@ -0,0 +1,40 @@
+#  conda env create -f scripts/mapd-deps-conda-dev-env.yml --force
```

Comment:
Replace this (out-dated) file with
https://github.com/Quansight/pearu-sandbox/blob/master/conda-envs/omniscidb-dev.yaml